packages/prismarine/src/inventory/ContainerEntry.ts

```diff
@@ -39,6 +40,9 @@ export default class ContainerEntry {
         stream.writeVarInt(0);
 
         // TODO: check for additional data
+        if (itemstack.getNetworkId() == item_id_map["minecraft:shield"]) {
```

Comment:
This should be `===` instead of `==`.
Also; use single quotes instead of double.

Reply:
Ok